playbooks/besman-criticality_score-playbook.sh

```diff
@@ -0,0 +1,92 @@
+function __besman_init {
+	echo -e "\ninit - collecting required info to execute the playbook\n"
+	echo -e "check & install snap"
+	if ! [ -x "$(command -v snap)" ]; then
+		echo "installing snap..."
+		sudo apt update
```

Comment:
@NeerajK007, make installations part of environment script. Only use playbook for execution